contributor/01-introduction-article.md

```diff
@@ -7,61 +7,60 @@ Perhaps after a while you even had to do some extra work in your project to work
-Such information could be established house rules for reviews, proposal of larger changes,  pointers to documentation or code sections relevant for your contribution.
+They are the gatekeepers to the project's code repository, and will move your contribution closer to production once they accepted it. 
+It is their role to mentor you on your way to contributing to their community. They may help you directly or provide information to enable you to help yourself. This information could be established house rules for reviews, proposal templates for larger changes, pointers to documentation or code sections relevant for your contribution.
 
-They also need to care about product quality, sustainability and project evolution from a technical as well as general perspective, about reducing the barrier to making to contributions for everyone as well as about caring for their community in general.
-Caring for the community involves keeping it healthy, upleveling it and its participants and advocating its needs in their organization.
+They also need to care about product quality, sustainability and project evolution from technical as well as general perspective, about reducing the barrier to making contributions for everyone, as well as about caring for their community in general.
+Caring for the community involves keeping it healthy, upleveling it and its participants, and advocating its needs in their organization.
 
 
 ### Product Owner
 The role of the [_Product Owner_][PO Introduction] has some similarity with the product owner role of your average project.
-However, there are some differences - depending on the size of the project, this role is often filled by the same person that acts as trusted committer.
-In larger projects or in teams that only partially use InnerSource as an approach to fill their needs by accepting contributions this role is likely filled by a person separate from a trusted committer.
+However, there are differences - depending on the size of the project, this role is often filled by the same person that acts as trusted committer.
+In larger projects, or in teams that only partially use InnerSource as an approach to fill their needs by accepting contributions, this role is likely filled by a person separate from a trusted committer.
 
-Your interaction with this role will likely focus on working out the fit of your contribution into the general product and its road map. 
-You might work this role to make sure that general aspects of documentation or consistency of UI/UX are kept upon merging your contribution.  
+Your interaction with product owner will likely focus on working out the fit of your contribution into the general product and its roadmap. 
+You might work with product owner to make sure that general aspects of documentation, or consistency of UI/UX, are kept upon merging your contribution.  
```

Comment:
```suggestion
You might work with the product owner role to make sure that general aspects of documentation, or consistency of UI/UX, are kept upon merging your contribution.  
```

Reply:
Adding this since Product Owner is not a person but a role. In many InnerSource projects, especially smaller ones you will find PO+TC filled by one person.